src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -1328,11 +1365,24 @@ private static object SearchIntrinsicTable(Hashtable table, Type callingType)
 
                 if (type != null)
                 {
-                    hashEntry = CreateInstance(type, callingType);
-                    if (type.GetConstructor(s_typeConstructor) == null)
+                    if (ConverterConstructorFuncs.TryGetValue(type, out Func<object> ctorFunc))
```

Comment:
This isn't correct because this method (`SearchIntrinsicTable`) is used by the "editors" as well. See 

[https://github.com/dotnet/runtime/blob/ea96ee3d86366ba21a5919101f64cc093a762fce/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.ReflectedTypeData.cs#L345-L352](https://github.com/dotnet/runtime/blob/ea96ee3d86366ba21a5919101f64cc093a762fce/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.ReflectedTypeData.cs#L345-L352)

Reply:
I added a new method for looking up intrinsic converters which helps maintain existing functionality for looking up "editors".